Routing/Loader/Reader/RestActionReader.php

```diff
@@ -352,16 +352,16 @@ private function getCondition(\ReflectionMethod $method, RouteAnnotation $annota
         $condition = $annotation->getCondition();
 
         if (!empty($this->versions)) {
-            $versionCondition = "request.attributes.get('version') == (";
+            $versionCondition = "request.attributes.get('version') in [";
             $first = true;
             foreach ($this->versions as $version) {
```

Comment:
I don't see how this loop could generate something like you mentioned in https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1436

`((<version> or <version>) or <version>)`